neptune/new/project.py

```diff
@@ -157,3 +157,204 @@ def _as_list(
+            wait (bool, optional): If `True` the client will first wait to send all tracked metadata to the server.
+                This makes the call synchronous. Defaults to `True`.
+
+        Examples:
+            >>> import neptune.new as neptune
+            >>> project = neptune.init_project(name='MY_WORKSPACE/MY_PROJECT')
+
+            >>> # Assign multiple fields from a dictionary
+            ... params = {"max_epochs": 10, "optimizer": "Adam"}
+            >>> project["parameters"] = params
+
+            >>> # You can always log explicitely parameters one by one
+            ... project["parameters/max_epochs"] = 10
+            >>> project["parameters/optimizer"] = "Adam"
+
+            >>> # Dictionaries can be nested
+            ... params = {"train": {"max_epochs": 10}}
+            >>> project["parameters"] = params
+            >>> # This will log 10 under path "parameters/train/max_epochs"
+
+        You may also want to check `assign docs page`_.
+
+        .. _assign docs page:
+            https://docs.neptune.ai/api-reference/project#.assign
+        """
+        AttributeContainer.assign(self, value=value, wait=wait)
+
+    def fetch(self) -> dict:
+        """Fetch values of all non-File Atom fields as a dictionary.
+        The result will preserve the hierarchical structure of the projects's metadata,
+        but will contain only non-File Atom fields.
+        You can use this method to quickly retrieve projects's parameters.
+
+        Returns:
+            `dict` containing all non-File Atom fields values.
+
+        Examples:
+            >>> import neptune.new as neptune
+            >>> project = neptune.init_project(name='MY_WORKSPACE/MY_PROJECT')
+            >>> params = project['model/parameters'].fetch()
+
+            >>> project_data = project.fetch()
+
+            >>> print(project_data)
+            >>> # this will print out all Atom attributes stored in project as a dict
+
+        You may also want to check `fetch docs page`_.
+
+        .. _fetch docs page:
+            https://docs.neptune.ai/api-reference/project#.fetch
+        """
+        return AttributeContainer.fetch(self)
+
+    def stop(self, seconds: Optional[Union[float, int]] = None) -> None:
+        """Stops the connection to the project and kills the synchronization thread.
+
+        `.stop()` will be automatically called when a script that initialized the connection finishes
+        or on the destruction of Neptune context.
+
+        When using Neptune with Jupyter notebooks it's a good practice to stop the connection manually as it
+        will be stopped automatically only when the Jupyter kernel stops.
+
+        Args:
+            seconds (int or float, optional): Seconds to wait for all tracking calls to finish
+                before stopping the tracked run.
+                If `None` will wait for all tracking calls to finish. Defaults to `True`.
+
+        Examples:
```

Comment:
Examples should be based on:
https://docs.neptune.ai/api-reference/project#.stop